From 9275c72c1a06c23119a26925f3c80519d4718b68 Mon Sep 17 00:00:00 2001
From: Bin Meng <bmeng.cn@gmail.com>
Date: Thu, 25 Jul 2019 19:04:04 -0700
Subject: [PATCH 1/1] doc: board: Add missing board index.rst files

These board index.rst files are missing in previous html doc patch
series, hence it causes some build warnings. Add those files.

Reported-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
Signed-off-by: Bin Meng <bmeng.cn@gmail.com>
Reviewed-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 doc/board/AndesTech/index.rst | 10 ++++++++++
 doc/board/atmel/index.rst     |  9 +++++++++
 doc/board/freescale/index.rst |  9 +++++++++
 doc/board/renesas/index.rst   | 10 ++++++++++
 doc/board/sifive/index.rst    |  9 +++++++++
 doc/board/xilinx/index.rst    |  9 +++++++++
 6 files changed, 56 insertions(+)
 create mode 100644 doc/board/AndesTech/index.rst
 create mode 100644 doc/board/atmel/index.rst
 create mode 100644 doc/board/freescale/index.rst
 create mode 100644 doc/board/renesas/index.rst
 create mode 100644 doc/board/sifive/index.rst
 create mode 100644 doc/board/xilinx/index.rst

diff --git a/doc/board/AndesTech/index.rst b/doc/board/AndesTech/index.rst
new file mode 100644
index 0000000000..d8f7d155fc
--- /dev/null
+++ b/doc/board/AndesTech/index.rst
@@ -0,0 +1,10 @@
+.. SPDX-License-Identifier: GPL-2.0+
+
+Andes Tech
+==========
+
+.. toctree::
+   :maxdepth: 2
+
+   adp-ag101p
+   ax25-ae350
diff --git a/doc/board/atmel/index.rst b/doc/board/atmel/index.rst
new file mode 100644
index 0000000000..8ba00fc227
--- /dev/null
+++ b/doc/board/atmel/index.rst
@@ -0,0 +1,9 @@
+.. SPDX-License-Identifier: GPL-2.0+
+
+Atmel
+=====
+
+.. toctree::
+   :maxdepth: 2
+
+   at91ek
diff --git a/doc/board/freescale/index.rst b/doc/board/freescale/index.rst
new file mode 100644
index 0000000000..8d42b35b96
--- /dev/null
+++ b/doc/board/freescale/index.rst
@@ -0,0 +1,9 @@
+.. SPDX-License-Identifier: GPL-2.0+
+
+Freescale
+=========
+
+.. toctree::
+   :maxdepth: 2
+
+   b4860qds
diff --git a/doc/board/renesas/index.rst b/doc/board/renesas/index.rst
new file mode 100644
index 0000000000..34e62baff6
--- /dev/null
+++ b/doc/board/renesas/index.rst
@@ -0,0 +1,10 @@
+.. SPDX-License-Identifier: GPL-2.0+
+
+Renesas
+=======
+
+.. toctree::
+   :maxdepth: 2
+
+   sh7752evb
+   sh7753evb
diff --git a/doc/board/sifive/index.rst b/doc/board/sifive/index.rst
new file mode 100644
index 0000000000..ad614c9bf2
--- /dev/null
+++ b/doc/board/sifive/index.rst
@@ -0,0 +1,9 @@
+.. SPDX-License-Identifier: GPL-2.0+
+
+SiFive
+======
+
+.. toctree::
+   :maxdepth: 2
+
+   fu540
diff --git a/doc/board/xilinx/index.rst b/doc/board/xilinx/index.rst
new file mode 100644
index 0000000000..2416fbd216
--- /dev/null
+++ b/doc/board/xilinx/index.rst
@@ -0,0 +1,9 @@
+.. SPDX-License-Identifier: GPL-2.0+
+
+Xilinx
+======
+
+.. toctree::
+   :maxdepth: 2
+
+   zynq
-- 
2.20.1

